L1Trigger/L1TTwinMux/src/RPCtoDTTranslator.cc

```diff
@@ -25,17 +25,28 @@
 
 using namespace std;
 
-
-RPCtoDTTranslator::RPCtoDTTranslator(RPCDigiCollection inrpcDigis){
-
-  m_rpcDigis = inrpcDigis;
-
+RPCtoDTTranslator::RPCtoDTTranslator(RPCDigiCollection const& inrpcDigis) :
+  m_rpcDigis{inrpcDigis}
+{
 }
 
 namespace {
   constexpr int max_rpc_bx = 2; 
   constexpr int min_rpc_bx = -2;
 
+  struct rpc_hit
```

Comment:
The struct is only used within the implementation so it is better to encapsulate it here.